spaceflights-pandas-viz/{{ cookiecutter.repo_name }}/pyproject.toml

```
@@ -37,6 +37,7 @@ namespaces = false
 package_name = "{{ cookiecutter.python_package }}"
 project_name = "{{ cookiecutter.project_name }}"
 kedro_init_version = "{{ cookiecutter.kedro_version }}"
+add_ons = {{ cookiecutter.add_ons | string | replace("\'", "\"") }}
```

In core template I've changed add_ons in `pyproject.toml`:
```suggestion
add_ons = "{{ cookiecutter.add_ons | default('') | string | replace('\"', '\\\"') }}"
```

Not sure if this was correct approach?

---

The ensures the default value is an empty string if no value is provided.